chore: remove filtering by username (DET-7095) (#4052)
determined-ai/determined: docs/release-notes/4052-filter-by-user.txt

:orphan:

**Breaking Changes**

-  API: Endpoints for getting lists of experiments, models, commands, notebooks, shells, and

>    tensorboards now filter by ``userIds`` instead of ``users`` (usernames)

Reviewer: ```suggestion
   tensorboards now filter by ``userIds`` instead of ``users`` (usernames),
```

:orphan:

**Breaking Changes**

> -  API: Endpoints for getting a list of experiments, models, commands, notebooks, shells, and
>    tensorboards now accept a ``userIds`` field of user IDs instead of a ``users`` field of usernames
>    for filtering by user.

Reviewer: ```suggestion
-  API: Endpoints for getting lists of experiments, models, commands, notebooks, shells, or
   tensorboards are now filterable by ``userIds`` instead of ``users``.
```
maybe just slightly clearer, but I'm bad a grammar.
